spec/models/event_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe Event, type: :model do
```

Comment:
i like that you started thinking about testing your `Event` here. i'd like to see a more robust test here and a test for the service you created 👍